src/view/zoom.ts

```diff
@@ -88,6 +88,8 @@ export class Zoom {
     }
 
     dblclick(e: MouseEvent) {
+        if(e.target !== e.currentTarget) return
+
```

Comment:
The problem with this solution is that in some cases, non-interactive elements can be added, and zoom shouldn't be prevented on double-click

Reply:
Hi @Ni55aN
Thank you for the response.
That's good to know, It's tricky for sure.
I will close the pull request.